app/views/hearing_days/show.html.haml

```diff
@@ -82,3 +82,9 @@
         = t('.event.none')
 
     = render partial: 'hearings/court_applications_v2', locals: { hearing: @hearing_details.hearing }
+
+    - if @application.subject_summary.proceedings_concluded
```

Comment:
Is there a case for changing this to @application&.subject_summary&.proceedings_concluded to make it safer and avoid potential nil errors?

Reply:
I dunno, if `@application` is nil here, or if `@application.subject_summary` doesn't return a SubjectSummary, then that's a sign something serious has gone wrong (and in practice this view would have blown up 40 lines higher up). To me that feels like a "this page is broken and we shouldn't try to  recover and show what would definitely be misleading data" kind of scenario.